CHANGELOG.md

```diff
@@ -57,17 +64,26 @@
 
 ### Breaking changes
 
+
 The `monaco-editor` dependency in `grafana-ui` has been updated to a newer version (`0.27.0`), which is not completely backward compatible with the old version (`0.21.2`). The backward incompatible changes are fairly small, but they do exist, so if your code accesses the raw monaco-objects through the `grafana-ui` package, please check the [monaco-editor changelog](https://github.com/microsoft/monaco-editor/blob/main/CHANGELOG.md) and apply any necessary changes. Issue [#39027](https://github.com/grafana/grafana/issues/39027)
 
-The mandatory `css` prop in `grafana/ui` components has been removed.
+
+The mandatory `css`  prop in `grafana/ui` components has been removed. 
```

Comment:
```suggestion
The mandatory `css` property in `grafana/ui` components has been removed. 
```
Write a complete word or term.

Reply:
updated this text in the source PR https://github.com/grafana/grafana/pull/38078